crates/nostr/src/nips/nip05.rs

```diff
@@ -5,10 +5,12 @@
 //!
 //! <https://github.com/nostr-protocol/nips/blob/master/05.md>
 
+use bitcoin_hashes::hex::ToHex;
 #[cfg(not(target_arch = "wasm32"))]
 use std::net::SocketAddr;
 use std::str::FromStr;
 
+use crate::Profile;
```

Comment:
```rust
#[cfg(feature = "base")]
use crate::Profile;
```

This piece of code should be under external crates: in my code I prefer to import `std`, `external crates` and  than `local modules` (in this order). Sorry for nitpicking.
I added `rustfmt.toml` file but the feature to order imported modules in not stable.